app/views/bills/_bill.html.erb

```diff
@@ -0,0 +1,15 @@
+<div id="<%= dom_id bill %>" class= 'bills-list'>
+  <div class= 'bill-icon'>
+    <%= image_tag bill.icon, alt: "Bill icon", class: 'icon' %>
```

Comment:
- [ ] Kindly provide a default image path that can be used when the user input an invalid icon URL or path.this is resulting to an error in your app, even in production. currently the deployed app is down because when I tried to create a transaction and add an invalite icon link, the app crushed. Kindly solve that

<img width="661" alt="image" src="https://github.com/Uthmanbello/budget_app/assets/65068771/3ea51123-3812-4a8f-a40e-d25f797531d7">
<img width="661" alt="image" src="https://github.com/Uthmanbello/budget_app/assets/65068771/49a35966-0f3a-4b5c-a695-0c75b055359d">

Reply:
Well resolved ✔️

![image](https://github.com/Uthmanbello/budget_app/assets/38088818/6ba725a4-d606-4715-aa90-b9d29d797b92)

Reply:
@KDlamini ,

Thanks a lot